classes/rawemailmessage.class.inc.php

```diff
@@ -223,10 +231,17 @@ public function GetAttachments(&$aAttachments = null, $aPart = null, &$index = 1
 				{
 					$sType = $aMatches[1];
 				}
-				if (empty($sFileName) && preg_match('/name="([^"]+)"/', $sContentType, $aMatches))
+				if (empty($sFileName) && preg_match('/name='.$sFileNameRegex.'/', $sContentType, $aMatches))
 				{
 					$sFileName = $aMatches[1];
 				}
+				//	Note: Since the RFC2045-compliant regex above has different levels of matches, 
+				// 		because of the use of capture groups, indexing into the results with 
+				//		$aMatches[1] sometimes returns the filename with quotes, but sometimes not. 
+				// 		For that reason (and for the sake of codesimplicity) we strip all quotes 
+				// 		from filenames here and will be fine.
+				$sFileName = str_replace("'", '', $sFileName);
+				$sFileName = str_replace('"', '', $sFileName);
```

Comment:
```suggestion
				$sFileName = trim($sFileName, '"\'');
```

But only when not implementing `$sFileName = last($aMatches);` as in that case, these can be removed..

Reply:
Tried it just now. Test results are identical. Your suggestion seems to be better. Will push.

![image](https://user-images.githubusercontent.com/21227916/207530716-e681b956-456c-4711-bfca-48e80eb9ca73.png)

Reply:
Closing conversation !